controller/integration/test/controlflow/controlflow_test.go

```diff
@@ -6,28 +6,29 @@ import (
 	"reflect"
 	"testing"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	"github.com/cenkalti/backoff"
 	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
 	"github.com/giantswarm/microerror"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
 	"github.com/giantswarm/operatorkit/controller"
 	"github.com/giantswarm/operatorkit/controller/integration/testresource"
 	"github.com/giantswarm/operatorkit/controller/integration/wrapper"
 	"github.com/giantswarm/operatorkit/controller/integration/wrapper/nodeconfig"
-	"k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/api/meta"
 )
 
-// Test_Finalizer_Integration_Controlflow is a integration test to
-// check that errors during deletion prevent the finalizer from removal.
+// Test_Finalizer_Integration_Controlflow is an integration test to check that
+// errors during deletion prevent the finalizer from removal.
```

Comment:
This here is just some boyscouting when I initially got my hands on the matter in the very beginning.